CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.7.0 2021-03-18
+
+* Provides fully tested functionality to register generic imaging data, with OMERO server support (v5.4.10)
```

Comment:
It's always nicer to link the PRs in the changelog which introduced the functionality 🔗

Reply:
Sounds good, I will add a link in the changelog :)